### PR TITLE
Display team-member's IRC nicks on hover.

### PR DESCRIPTION
**Pull request by huonw**
https://internals.rust-lang.org/t/announcing-the-subteams/2042/2?u=huon

Embarrassingly, I'm unsure of @jakub-'s exact IRC nick.

You can see what this looks like at: http://huonw.github.io/rust-www/team.html

### CODE REVIEW

File: team.md [hunk 1]
```diff
@@ -1,20 +1,118 @@
...
+  "jakub-":
+    name: Jakub Bukaj
+  Kimundi:
+    name: Marvin Löbel
+    irc: kimundi
+  manishearth:
+    name: Manish Goregaokar
+    irc: Manishearth
+  mbrubeck:
+    name: Matt Brubeck
```
Comment (Manishearth) [inline team.md]: irc: mbrubeck

File: team.md [hunk 2]
```diff
@@ -1,20 +1,118 @@
...
+  michaelwoerister:
+    name: Michael Woerister
+    irc: mw
+  nikomatsakis:
+    name: Niko Matsakis
+    irc: nmatsakis
+  nrc:
+    name: Nick Cameron
+  pcwalton:
+    name: Patrick Walton
```
Comment (Manishearth) [inline team.md]: irc: pcwalton

**Comment by huonw** [comment]
(Rebased.)

**Comment by alexcrichton** [comment]
Nice!

**Pull request merged**